snuba/datasets/transactions.py

```diff
@@ -121,7 +131,9 @@ def __init__(self):
                 ("sdk_name", WithDefault(String(), "''")),
                 ("sdk_version", WithDefault(String(), "''")),
                 ("tags", Nested([("key", String()), ("value", String())])),
+                ("tags_map", String()),
```

Comment:
I think this name is a bit misleading — I'd say the existing `tags` is a better approximation of a map than this column. What about `tags_flattened`?

Should we consider prefixing these definitely-not-public fields with a leading underscore?

Reply:
I can change the name here, the issue is that the column is already in prod, with this name :(

Reply:
I'm honestly ok with changing the name to whatever, just change the DDL and tell me what it should be. We don't use the current name and it's easily dropped.

Reply:
ok, let me collect all the changes we need, so we can do one migration only in prod